doc: add minutes for meeting 31 Jul 2024 (#1604)
nodejs/TSC: meetings/2024-07-31.md

    source code that we have in the repo. Could result in supply chain attack
    * proposing, that we add check that wasm is as expected, could be a short term solution until
      we improve the build process.
  * Marco - wasm is built in the same way that swc builds the package. It uses the crates
    released in the rust registry. So same issue would apply to other swc users. Similar to saying
    that everything which pulls package from npm is vulnerable as well.
    * We can fix with a lock file, and should do that.
    * Not sure what adding a check will fix, and it applies to other ways that we build
      dependencies as well.
  * Nikita, there are some other fixes that should be pulled in in addition to adding the security
    check.
  * Matteo, don’t think the safeguard is needed, since --experimental-strip-types is not run by
    default. For those who are trying out the feature, looking at the risk, don’t see a significant risk
    in this specific moment. Agree that to unflag it should be a top priority. This path is no the
    highest risk, so not necessarily the place to focus.

>   * Antoine, as long as it stays dev only, runtime check is ok if it is off thread. Maybe move off

Reviewer: ```suggestion
  * Antoine, as long as it stays dev only, runtime check is ok. Maybe we can move the check off
```
